ash/src/vk/definitions.rs

```diff
@@ -1454,6 +1461,17 @@ pub struct PhysicalDeviceMemoryProperties {
     pub memory_heap_count: u32,
     pub memory_heaps: [MemoryHeap; MAX_MEMORY_HEAPS],
 }
+#[cfg(feature = "debug")]
```

Comment:
Unintended changes here and below?

Reply:
Looks like I stil had the `len=` annotation from testing #858